.github/workflows/quality-check.yml

```
@@ -91,12 +91,25 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     if: ${{ github.event_name == 'pull_request' }}
+    strategy:
+      matrix:
+        ps_version: [ "1.7.8.10", "8.1.1", "latest" ]
     steps:
       - name: Checkout the repository
         uses: actions/checkout@v4
         with:
           ref: ${{ github.ref }}
 
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
```

```suggestion
      - uses: pnpm/action-setup@v3
        with:
          version: latest
```
Or just a `npm install -g pnpm`

---

ok